File: comms/bulk-messaging/messages/send-message/send-message.py
from twilio_comms import TwilioComms

client = TwilioComms(
    account_id="TWILIO_ACCOUNT_SID",
    auth_token="TWILIO_AUTH_TOKEN",
)

client.messages.send(
    to=[
        {
            "address": "+12065558844",
            "channel": "PHONE"
        }
    ],
    content={
        "text": "Hello, World!"
    },
    from_={
        "address": "+14153901002",
        "channel": "SMS"
    },
)

File: comms/bulk-messaging/messages/send-message/send-message.cs
using Twilio.Comms;
using System.Threading.Tasks;
using System.Collections.Generic;
using OneOf;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.Messages.SendAsync(
            new MessagesSendRequest {
                To = new List<OneOf<MessagesSendRequestToItemAddress, MessagesSendRequestToItemAddresses, MessagesSendRequestToItemMemoryStoreId>>(){
                    new MessagesSendRequestToItemAddress {
                        Address = "+12065558844",
                        Channel = MessageAddressChannel.Phone
                    },
                }
                ,
                Content = new MessageContentTextWithMedia {
                    Text = "Hello, World!"
                },
                From = new MessageAddressSender {
                    Address = "+14153901002",
                    Channel = MessageSenderChannel.Sms
                }
            }
        );
    }

}
